### DISCUSSION TITLE
examples/trackPlayer/

### DISCUSSION THREAD

**Discussion by giscus** [vaebe/leafletjsExample#19] [Q&A]
# examples/trackPlayer/

leafletjs 与 vue3 结合使用的一些示例

http://localhost:5173/examples/trackPlayer/

**Comment by vaebe** [comment]
这是第一个评论

**Comment by jxh-0514** [comment]
地图坐标系为L.CRS.simple的时候传人x,y的轨迹数据，播放时会有问题。

**Reply by vaebe** [reply]
leafletjs 的默认坐标系是 `L.CRS.EPSG3857` [官方网站查看具体配置](https://leafletjs.cn/reference.html#map-option)

传入的数据坐标系需要与 leafletjs 设置的坐标系保持一致
